### PR TITLE
added new variants for Chambers and Gift Card Reload Card

### PR DESCRIPTION
**Pull request by Sukumar-K7**
authentication.nudge.alert-message

image

  {
        title: "[IHCL Forms] Dining Plan An Event Enquiry",
        value: "ihclForms.dining-plan-an-event-enquiry",
      }
image

     {
        title: "[IHCL Forms] Description with CTA",
        value: "ihclForms.group.table-reservation",
      },
image

### CODE REVIEW

File: schemas/chambers/index.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { FeatureSchemaDefinition } from "schemas/types";
+
+export const chambers: FeatureSchemaDefinition = {
+  variants: {
+    card: [
+      {
+        title:
+          "[Chambers] Vertical Left Media With Graphical Bullet Line With Respect To Right Content",
+        value:
+          "chambers.card.vertical-left-media-with-graphical-bullet-line-with-respect-to-right-content",
```
Comment (NarsingRao13) [inline schemas/chambers/index.ts:10]: Too long for variant name

File: schemas/chambers/index.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      },
+      {
+        title:
+          "[Chambers] Vertical Right Media With Graphical Bullet Line With Respect To Left Content",
+        value:
+          "chambers.card.vertical-right-media-with-graphical-bullet-line-with-respect-to-left-content",
```
Comment (NarsingRao13) [inline schemas/chambers/index.ts:16]: Too long for variant name

Comment (Sukumar-K7) [reply schemas/chambers/index.ts:16]: noted and will changge

File: schemas/chambers/index.ts (hunk 1 above)
Comment (Sukumar-K7) [reply schemas/chambers/index.ts:10]: okay I will changge

File: schemas/commonUtils/index.ts [hunk 3]
```diff
@@ -216,6 +214,10 @@ export const ihclcore: FeatureSchemaDefinition = {
         title: "Group With SignUp Form",
         value: "ihcl.core.group.signup-form",
       },
+      {
+        title: "Cookie Management System Modal",
+        value: "ihcl.core.group.cookie-management-system-modal",
```
Comment (NarsingRao13) [inline schemas/commonUtils/index.ts:219]: Going further please remove **ihcl.core**. Instead use **common** or **commonutils**

File: schemas/commonUtils/index.ts [hunk 4]
```diff
@@ -350,5 +352,11 @@ export const ihclcore: FeatureSchemaDefinition = {
         value: "ihcl.core.switchCaseBlock.payments",
       },
     ],
+    categoryHighlights: [
+      {
+        title: "category with logo and text",
+        value: "ihcl.core.categoryHighlights.category-with-logo-and-text",
```
Comment (NarsingRao13) [inline schemas/commonUtils/index.ts:358]: Same here also

File: schemas/commonUtils/index.ts (hunk 3 above)
Comment (Sukumar-K7) [reply schemas/commonUtils/index.ts:219]: okay

File: schemas/commonUtils/index.ts (hunk 4 above)
Comment (Sukumar-K7) [reply schemas/commonUtils/index.ts:358]: okay

File: schemas/chambers/index.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { FeatureSchemaDefinition } from "schemas/types";
+
+export const chambers: FeatureSchemaDefinition = {
+  variants: {
+    card: [
+      {
+        title:
+          "[Chambers] Left Vertical Media With Bullet Line and Right Content",
```
Comment (NarsingRao13) [inline schemas/chambers/index.ts:8]: Can't have any other nomenclature. Which should be more generic.

Comment (Sukumar-K7) [reply schemas/chambers/index.ts:8]: can it be somthing like "vertical oriented Left media", "horizonatal oriented Left media"

Comment (NarsingRao13) [reply schemas/chambers/index.ts:8]: Make it specific.

**Review by NarsingRao13 — APPROVED** [review]

**Pull request merged**